glide.yaml

```diff
@@ -3,7 +3,7 @@ import:
 - package: github.com/Masterminds/goutils
   version: ^1.0.0
 - package: github.com/satori/go.uuid
-  version: ^1.1.0
+  version: =1.2.0
```

Comment:
A nit but can use use `^1.2.0`. If they do go back and release a bug fix or some other release based on 1.2 we will pick it up. May be unlikely but better safe than sorry.

Reply:
I did this on purpose to avoid the same problem that triggered me into making this pull request occuring again, but I guess you're right. I'll change it.